pkg/apis/core/types.go

```diff
@@ -488,6 +488,15 @@ type PersistentVolumeClaimSpec struct {
 	// (Alpha) Using the namespace field of dataSourceRef requires the CrossNamespaceVolumeDataSource feature gate to be enabled.
 	// +optional
 	DataSourceRef *TypedObjectReference
+	// volumeAttributesClassName is the name of the VolumeAttributesClass required by the claim.
+	// If specified, the csi-driver will create or update the volume with the attributes defined
+	// in the corresponding VolumeAttributesClass. Different with storageClassName, it can be
+	// changed after the claim is created but an empty string value is disallowed.
+	// More info: https://kubernetes.io/docs/concepts/storage/persistent-volumes#volumeattributesclass
```

Comment:
I'm not sure that we can refer to this doc link that does not exist yet: https://kubernetes.io/docs/concepts/storage/persistent-volumes#volumeattributesclass

Reply:
@msau42 what do you think ? 

remove the link or add a TODO word ?